### DISCUSSION TITLE
How to create remote actor  dynamically

### DISCUSSION THREAD

**Discussion by zuston** [tqwewe/kameo#101] [Q&A]
Is this possible to support creating remote actor dynamically like the ray did? https://github.com/ray-project/ray

**Comment by tqwewe** [comment]
I'm not familiar with ray. I'm not entirely sure what is meant by creating a remote actor dynamically? Could you provide some context or an example?

**Reply by zuston** [reply]
> I'm not familiar with ray. I'm not entirely sure what is meant by creating a remote actor dynamically? Could you provide some context or an example?

Simply to say, ray can create a customize actor in client side, and then it can be deployed in the remote cluster. From this point, the node cluster as the execution pool, and the actor can be created after this, that there is no need to define actor before cluster starting. 

Feel free to discuss more

**Reply by zuston** [reply]
https://docs.ray.io/en/latest/ray-core/actors.html

**Reply by tqwewe** [reply]
I'm not entirely clear still on the concept, but from what I understand, Kameo doesn't support this. Currently, actors can only be spawned locally, and any node which wants to communicate with it must be aware of its type
